prog/prog.go

```diff
@@ -5,12 +5,16 @@ package prog
 
 import (
 	"fmt"
+	"github.com/google/syzkaller/pkg/mab"
```

Comment:
Empty line between standard and non-standard packages please. These will be sorted separately.